xconfess-backend/.env.sample

```
@@ -1,5 +1,52 @@
+CONFESSION_AES_KEY=12345678901234567890123456789012
+
+# ──────────────────────── Redis ──────────────────────────
+REDIS_HOST=localhost
+REDIS_PORT=6379
+# REDIS_PASSWORD=             # optional — leave blank for local
+
+# ──────────────────────── Email (Nodemailer / SMTP) ──────
+EMAIL_HOST=smtp.example.com
+EMAIL_PORT=587
+EMAIL_USER=your_email@example.com
+EMAIL_PASS=your_email_password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_PASS key should go before the EMAIL_PORT key

(UnorderedKey)